### PR TITLE
[C#] replace conditionals with if-statements

### PR DESCRIPTION
**Pull request by valentin-p**
From @ErikSchierboom 
We should make it consistent. I think for now changing if-statements to conditionals makes the most sense.

### CODE REVIEW

**Review by mikedamay — COMMENTED** [review]
Can you have a look at other mentions of "if-statements" in the C# directory tree such as _design.md_ documents.  This is slightly complicated by the presence of automatically generated documents such as _csharp/reference/README.md_ but I suspect it doesn't matter if you modify these even though that is not necessary.

**Comment by valentin-p** [comment]
Error: [C#] languages/csharp/concepts/conditionals/about.md is missing
Error: [C#] languages/csharp/concepts/conditionals/links.json is missing

**Comment by ErikSchierboom** [comment]
@valentin-p Having seen the PR changes, I'm now thinking I might actually prefer `if-statements` over `conditionals` 🤦 

I'd be happy to merge though and change it later though, if you'd like to get this merged. Sorry about this!

File: languages/csharp/reference/exercises.json [hunk 1]
```diff
@@ -349,7 +349,7 @@
           ]
         },
         {
-          "name": "if-statements",
+          "name": "conditionals",
```
Comment (ErikSchierboom) [inline languages/csharp/reference/exercises.json:352]: This change should be reverted I think

File: languages/csharp/reference/README.md [hunk 2]
```diff
@@ -372,7 +372,7 @@ This report is updated automatically when any _design.md_ document is modified o
 [tnc-foreach-loops]: ../../../reference/concepts/enumeration.md
 [tnc-functions]: ../../../reference/types/function.md
 [tnc-higher-order-functions]: ../../../reference/concepts/higher_order_functions.md
-[tnc-if-statements]: ../../../reference/concepts/conditionals.md
+[tnc-conditionals]: ../../../reference/concepts/conditionals.md
```
Comment (ErikSchierboom) [inline languages/csharp/reference/README.md:375]: This change should be reverted I think

File: languages/csharp/reference/README.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ This report is updated automatically when any _design.md_ document is modified o
 - foreach-loops [_(bird-watcher)_][exercise-bird-watcher] - [Design][design-bird-watcher], [Background][tnc-foreach-loops]
 - functions [_(lucians-luscious-lasagna)_][exercise-lucians-luscious-lasagna] - [Design][design-lucians-luscious-lasagna], [Background][tnc-functions]
 - generic-types [_(tracks-on-tracks-on-tracks)_][exercise-tracks-on-tracks-on-tracks] - [Design][design-tracks-on-tracks-on-tracks]
-- if-statements [_(cars-assemble)_][exercise-cars-assemble] - [Design][design-cars-assemble], [Background][tnc-if-statements]
+- conditionals [_(cars-assemble)_][exercise-cars-assemble] - [Design][design-cars-assemble], [Background][tnc-conditionals]
```
Comment (ErikSchierboom) [inline languages/csharp/reference/README.md:51]: This change should be reverted I think

File: languages/csharp/exercises/concept/squeaky-clean/.meta/design.md [hunk 4]
```diff
@@ -25,5 +25,5 @@
 ## Prerequisites
 
 - `strings`: know of the `string` type that will be iterated over and accessed by index.
-- `for-loop` for loops (rather than foreach) are the best means of highlighting the relationship between strings and `char`s
-- `if-statements`:
+- `for-loop`: for loops (rather than foreach) are the best means of highlighting the relationship between strings and `char`s
```
Comment (ErikSchierboom) [inline languages/csharp/exercises/concept/squeaky-clean/.meta/design.md:28]: Just one thing I noticed while reviewing, this concept should be `for-loops`. We should fix this in a separate PR.

File: languages/csharp/exercises/concept/cars-assemble/.docs/introduction.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ C# has two types of numeric conversions:
 
 As an `int` has less precision than a `double`, converting from an `int` to a `double` is safe and is thus an implicit conversion. However, converting from a `double` to an `int` could mean losing data, so that requires an explicit conversion.
 
-## if-statements
+## conditionals
```
Comment (ErikSchierboom) [inline languages/csharp/exercises/concept/cars-assemble/.docs/introduction.md:19]: This change should be reverted I think

**Review by ErikSchierboom — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Comment by mikedamay** [comment]
LGTM.  @valentin-p really pleased you're involved.

> Having seen the PR changes, I'm now thinking I might actually prefer if-statements over conditionals 

@valentin-p Not sure of your nationality but do think this could be an aspect of the Dutch sense of humour.

**Pull request merged**